maps-app/src/main/java/com/esri/android/mapsapp/MapFragment.java

```diff
@@ -263,7 +257,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 				mBasemap = new Basemap(defaultBaseMapURL);
 				mMap = new ArcGISMap(mBasemap);
 
-        MapFragment.mMapView = (MapView) mMapContainer.findViewById(R.id.map);
+        MapFragment.mMapView = mMapContainer.findViewById(R.id.map);
```

Comment:
I don't think the `mMapView` needs to be static, this causes a leak too.

Reply:
I have no idea why it would be static either, I guess I can just drop it, retest everything and make sure there's not any weird reliance on it being static.